### PR TITLE
2825: Individual Camouflage Bulk Assignment from Same Original Camouflage

### LINKED ISSUE #2825: RFE [0.48] Change Individual Camo for a selection for Hangar Objects
### Environment
0.48
Windows 10
OpenJDK 11

### Description
In order to add individual camos to a group of mechs each one has to be done individually.  Adding a grouping function that lets you apply camo for some or all of them would be less time consuming. 
When changing forces in megamek to other players, only the individual camo is respected otherwise the camos changes.

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes #2825

### CODE REVIEW

File: MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java [hunk 1]
```diff
@@ -419,17 +419,19 @@ public void actionPerformed(ActionEvent action) {
...
+            final CamoChooserDialog ccd = new CamoChooserDialog(gui.getFrame(),
                     selectedUnit.getUtilizedCamouflage(gui.getCampaign()), true);
             if (ccd.showDialog().isCancelled()) {
                 return;
             }
-            selectedUnit.getEntity().setCamouflage(ccd.getSelectedItem());
-            MekHQ.triggerEvent(new UnitChangedEvent(selectedUnit));
+            for (final Unit unit : units) {
+                unit.getEntity().setCamouflage(ccd.getSelectedItem());
+                MekHQ.triggerEvent(new UnitChangedEvent(unit));
```
Comment (NickAragua) [inline MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java:433]: So here's a thought: why are we triggering the unit changed event here but not for the remove camo command? Also, why are we triggering it at all?

**Review by NickAragua — APPROVED** [review]
One thought unrelated to the content of the PR itself, but looks fine otherwise.

File: MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java (hunk 1 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java:433]: Nice catch... I'll double check if it is required, and either add or remove if it is. IIRC it is supposed to force the icon to update

Comment (Windchild292) [reply MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java:434]: It's a bug with the remove camo command. I've updated it for the future.

**Pull request merged**